src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="block">
+      <div class="calendar calendar--start-mon calendar--length-31">
```

Comment:
```suggestion
      <div class="calendar calendar--start-day--mon calendar--month-length--31">
```
Fix modifiers according to the task description
![image](https://github.com/mate-academy/layout_calendar/assets/96409563/e1cd6fe8-7d3f-4cec-bf87-82b43bd17143)
![image](https://github.com/mate-academy/layout_calendar/assets/96409563/e5f8fb45-54fb-43c7-8284-5dc8beac3e1e)